internal/commands/show_status.go

```diff
@@ -162,19 +161,6 @@ func createStatusAttachment(ctx context.Context, client bot.Client, logger log.L
 			Color:    "#f2b12e",
 			Fields:   fields,
 		}
-	} else {
-		field := slack.AttachmentField{
-			Title: config.Env.Messages.NoTimelineItems,
-		}
-		fields = append(fields, field)
-
-		attach = slack.Attachment{
-			Pretext:  "Incident Status:",
-			Fallback: "Incident Status",
-			Text:     "",
-			Color:    "#999999",
-			Fields:   fields,
-		}
```

Comment:
As we discuss this message can be removed, but I think if this scenario happens, we should at least log it.